planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md

```diff
@@ -604,31 +604,52 @@ namespace: `avoidance.`
 
 namespace: `avoidance.target_object.`
 
-| Name       | Unit | Type | Description                                | Default value |
-| :--------- | ---- | ---- | ------------------------------------------ | ------------- |
-| car        | [-]  | bool | Allow avoidance for object type CAR        | true          |
-| truck      | [-]  | bool | Allow avoidance for object type TRUCK      | true          |
-| bus        | [-]  | bool | Allow avoidance for object type BUS        | true          |
-| trailer    | [-]  | bool | Allow avoidance for object type TRAILER    | true          |
-| unknown    | [-]  | bool | Allow avoidance for object type UNKNOWN    | false         |
-| bicycle    | [-]  | bool | Allow avoidance for object type BICYCLE    | false         |
-| motorcycle | [-]  | bool | Allow avoidance for object type MOTORCYCLE | false         |
-| pedestrian | [-]  | bool | Allow avoidance for object type PEDESTRIAN | false         |
+This module supports all object classes, and it can set following parameters independently.
+
+```yaml
+car:
+  enable: true # [-]
+  moving_speed_threshold: 1.0 # [m/s]
+  moving_time_threshold: 1.0 # [s]
+  max_expand_ratio: 0.0 # [-]
+  envelope_buffer_margin: 0.3 # [m]
+  avoid_margin_lateral: 1.0 # [m]
+  safety_buffer_lateral: 0.7 # [m]
+  safety_buffer_longitudinal: 0.0 # [m]
+```
+
+| Name                       | Unit  | Type   | Description                                                                                                | Default value |
+| :------------------------- | ----- | ------ | ---------------------------------------------------------------------------------------------------------- | ------------- |
+| enable                     | [-]   | bool   | Allow avoidance for object type CAR                                                                        | false         |
+| moving_speed_threshold     | [m/s] | double | Vehicles with speed greater than this will be judged as moving ones.                                       | 1.0           |
```

Comment:
Vehicles -> target object?

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/4073/commits/c404fb8eb97dfcedea624e39175aa118b123d0fc.